Dockerfile

```diff
@@ -1,35 +1,29 @@
-FROM debian
-MAINTAINER Andreas Stenius git@astekk.se
+FROM alpine:3.4
+MAINTAINER Marco Wessel <marco@driebit.nl>
 
-ENV DEBIAN_FRONTEND noninteractive
-
-# httpredir.debian.org/debian fails too much, so replace it with a fixed mirror.
-RUN echo \
-   'deb ftp://ftp.nl.debian.org/debian/ jessie main\n \
-    deb ftp://ftp.nl.debian.org/debian/ jessie-updates main\n \
-    deb http://security.debian.org jessie/updates main\n' \
-    > /etc/apt/sources.list
+ADD . /opt/zotonic
+WORKDIR /opt/zotonic
 
-ADD https://packages.erlang-solutions.com/erlang-solutions_1.0_all.deb .
 
-RUN apt-get clean \
-    && dpkg -i erlang-solutions_1.0_all.deb \
-    && apt-get update \
-    && apt-get install -y --no-install-recommends erlang build-essential ca-certificates imagemagick inotify-tools libnotify-bin wget git \
-    && rm -rf /var/lib/apt/lists/*
+# Note: dumb-init and gosu are pulled from edge; remove that when upgrading to an alpine release that
+# includes those packages.
```

Comment:
Better safe than sorry. Running as root is generally not a good idea, regardless of whether or not you're in a container. Linux cgroups is not yet (and probably won't ever be) 100% secure.